wasmer/interface.go

```diff
@@ -29,3 +31,7 @@ type MemoryHandler interface {
 	Destroy()
 	IsInterfaceNil() bool
 }
+
+type ValidatorContext interface {
```

Comment:
add comment for interface

Reply:
IsInterfaceNil() bool

Reply:
Turns out it was unused, deleted altogether.